src/drivers/bitbucket_cloud.js

```diff
@@ -536,6 +546,10 @@ class BitbucketCloud {
 
     return responseBody;
   }
+
+  warn(message) {
+    console.error(message);
+  }
```

Comment:
```suggestion
```

Reply:
Perhaps use `winston.warn` instead?

```suggestion

  warn(message) {
    winston.warn(message);
  }
```

Reply:
Fixed in https://github.com/iterative/cml/pull/1241/commits/e7a7996b731956e891a45087a0d1ccad611ff6d7